src/controllers/favorites.controllers.js

```diff
@@ -21,3 +20,11 @@ export const addToFavorite = async (data) => {
     return { status: 'invalid', message: err.message };
   }
 };
+export const findAllFavorites = async () => {
+  try {
+    const favorites = await favoriteModel.find().skip(0).limit(10);
```

Comment:
This should return data of a user's favorite houses. It'd probably be best if data included fields just like the ones needed to display the house list on the index page as the purpose of this endpoint is to get data needed to display a similar list.